### PR TITLE
Add-DbaDbRoleMember - Add support for adding roles to roles

### LINKED ISSUE #9302: `Add-DbaDbRoleMember` - Unable to Add Role
### Verified issue does not already exist?

I have searched and found no existing issue

### What error did you receive?

```powershell
WARNING: [10:29:51][Add-DbaDbRoleMember] User MyRole does not exist in [database] on [sqlinstance]
```

### Steps to Reproduce

```powershell
New-DbaDbRole sqlinstance -Database database -Role MyRole
add-dbadbrolemember sqlinstance -Database database -Role db_datareader,db_denydatawriter -User MyRole
```

### Please confirm that you are running the most recent version of dbatools

```powershell
Major  Minor  Build  Revision
-----  -----  -----  --------
2      1      11     -1
```

### Other details or mentions

One could update this section of code to check for `$db.Roles.Name`.
https://github.com/dataplat/dbatools/blob/7af0e3aec6b9d621fa93af0e2ca5f1c621e7140c/public/Add-DbaDbRoleMember.ps1#L143

Maybe there needs to be another parameter called `DatabaseRole` much like `Add-DbaServerRoleMember`? I guess that precedent overrides the simpler solution of accepting both `Roles` and `Users` for `-User`. I could also see `-Principal` being a parameter instead though none of the cmdlets use that language, I think.

### What PowerShell host was used when producing this error

PowerShell Core (pwsh.exe)

### PowerShell Host Version

```powershell
Name                           Value
----                           -----
PSVersion                      7.3.7
PSEdition                      Core
GitCommitId                    7.3.7
OS                             Microsoft Windows 10.0.20348
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### SQL Server Edition and Build number

```sql
Microsoft SQL Server 2019 (RTM-CU18) (KB5017593) - 15.0.4261.1 (X64)
        Sep 12 2022 15:07:06
        Copyright (C) 2019 Microsoft Corporation
        Enterprise Edition: Core-based Licensing (64-bit) on Windows Server 2019 Datacenter 10.0 <X64> (Build 17763: ) (Hypervisor)
```

### .NET Framework Version

```powershell
.NET 7.0.11
```

### PR DESCRIPTION
**Pull request by andreasjordan**
## Type of Change

 - [x] Bug fix (non-breaking change, fixes #9302 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (affects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/dataplat/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

See issue for details.

### CODE REVIEW

**Comment by andreasjordan** [comment]
We have to change Get-DbaDbRoleMember as well. Will work on this in the next days.

**Comment by mattcargile** [comment]
The code looks good. Thanks!

Would love to see `DbaServerRoleMember` cmdlets get the same treatment on the parameters. Might involve more breaking changes.

**Comment by andreasjordan** [comment]
Let's first have @potatoqualitee have a look at this PR. Then we can talk about breaking changes.

**Comment by potatoqualitee** [comment]
thank you for the alias! @ClaudioESSilva can you pls take a look and approve?

File: public/Add-DbaDbRoleMember.ps1 [hunk 1]
```diff
@@ -22,8 +22,8 @@ function Add-DbaDbRoleMember {
     .PARAMETER Role
         The role(s) to process.
 
-    .PARAMETER User
-        The user(s) to add to role(s) specified.
+    .PARAMETER Member
+        The user(s) or role(s) to add to role(s) specified.
```
Comment (wsmelton) [inline public/Add-DbaDbRoleMember.ps1:26]:
```suggestion
        The member(s) (user or role) to add to the Roles specified.
```

**Review by wsmelton — CHANGES_REQUESTED** [review]
Just rewording suggesting...but LGTM

**Comment by potatoqualitee** [comment]
oh, good. thanks everyone 😊

**Pull request merged**